app/Http/Controllers/Users/BulkUsersController.php

```diff
@@ -219,7 +219,7 @@ public function destroy(Request $request)
 
         $users = User::whereIn('id', $user_raw_array)->get();
         $assets = Asset::whereIn('assigned_to', $user_raw_array)->where('assigned_type', \App\Models\User::class)->get();
-        $accessories = DB::table('accessories_users')->whereIn('assigned_to', $user_raw_array)->get();
+        $accessories = DB::table('accessories_checkout')->whereIn('assigned_to', $user_raw_array)->get();
```

Comment:
I think this query will also not work, since it's not constraining to users. Location ID could get pulled from that list as well as User ID 4.